frontend-v2/src/views/Intro.vue

```diff
@@ -37,24 +37,16 @@
     </template>
     <template slot="body">
       <h1>How it works</h1>
-      <p> 
-        The initial highlighted polygon will always represent your current location. You can tap on any of the displayed polygons to get information about the restrictions in the area. 
+      <p>
+        The initial highlighted polygon will always represent your current location. You can tap on any of the displayed polygons to get information about the restrictions in the area.
 
-        The for states of the locations are the following: 
+        The for states of the locations are the following:
       </p>
       <div class="chip-wrap">
-        <div class="chip mini open">
-          Open
-        </div>
-        <div class="chip mini private">
-          Private
-        </div>
-        <div class="chip mini conditional">
-          Conditional
-        </div>
-        <div class="chip mini unknown">
-          Unknown
-        </div>
+        <StatusButton theme="success" text="Open" ></StatusButton>
```

Comment:
Yeah. That was actually my first attempt. But then I came across errors in vue template since it wouldn't recognize the `OpenStatus` constant. I know there are work arounds to that. Let's review and discuss it next week when we meet. The reason I went for `theme`, `text`, and `size` properties for the component was to make it as generic as possible, but we probably don't need that and can use it only for statuses (after all, I named it StatusButton for a reason, right?)